server/location/index.js

```diff
@@ -0,0 +1,175 @@
+const debug = require('debug')('kyf:matcher');
+
+// const { constituencyFromPostcode, locationInfoFromPostcode } = require ('./externals');
+const { isPc7, isPc8, isPostcodeDistrict, isFullPostcode, isPostcode, endsWithPostcode,
+pc7FromFullPostcode, pc8FromFullPostcode, districtFromFullPostcode, districtFromPostcodeDistrict, postcodeFromString,
+toStandardLatLong, toLatLong, isLeafletLatLng, latLongFromString, latLongFrom4dpLatLongString, roundToNearest } = require ('./helpers');
+const { fromPostcodesIo, fromGoogle, fromTwitter,
+  constituencyFromPostcode, locationInfoFromPostcode  } = require ('./requests');
+
+const LocationObject = ()=> ({
```

Comment:
this could be a `class`, if you're using it like one

Reply:
I can't see the convincing argument for or against,
but, for now, have not implemented a class because plenty of stuff to do and I can't be 100% confident of no resulting unpredictable results to chase down.